packages/video_player/video_player/example/integration_test/video_player_test.dart

```diff
@@ -240,4 +240,73 @@ void main() {
       expect(fileController.value.isPlaying, false);
     }, skip: kIsWeb);
   });
+
+  group('asset audios', () {
+    setUp(() {
+      _controller = VideoPlayerController.asset('assets/Audio.mp3');
+    });
+
+    testWidgets('can be initialized', (WidgetTester tester) async {
+      await _controller.initialize();
+
+      expect(_controller.value.isInitialized, true);
+      expect(_controller.value.position, const Duration(seconds: 0));
+      expect(_controller.value.isPlaying, false);
+      // Due to the duration calculation accurancy between platforms,
+      // the milliseconds on Web will be a slightly different from natives.
+      // The audio was made with 44100 Hz, 192 Kbps CBR, and 32 bits.
+      if (kIsWeb) {
```

Comment:
It's 5.041625 sec. The web takes `5.042` and natives take `5.041`.

Reply:
I've done some debug steps, and the root cause of this difference is from here:
https://github.com/flutter/plugins/blob/a10ac2e0d8a8cd878fd0d0a54aa9a87c44c7d7c5/packages/video_player/video_player_web/lib/video_player_web.dart#L301
If we want to make them equal, we might use `toInt()` here.